### DISCUSSION TITLE
Windows 11 ARM64 feedback

### DISCUSSION THREAD

**Discussion by hudsonhu** [WhatTheBlock/WindowsSimplify#48] [General]
Thank you for your hard work!! Here are some feedback of arm64 version:

Windows 11 220829.iso cannot be installed, and stuck after one reboot, showing this:
<img width="708" alt="截屏2022-10-18 21 22 52" src="https://user-images.githubusercontent.com/48980398/196441798-06a15f00-c6dc-426a-a61c-e8b0786dcb33.png">

Version 220722 worked fine, tested in my Parallels Desktop 18 vm.

It would be great if you can release an ARM64 Windows 11 which removes Windows Update, I would be very happy to participate testing.

**Comment by WhatTheBlock** [comment]
22621.232_arm_220829 has removed windows update.
The bug of specialize has been fixed and the file is uploading.

**Comment by WhatTheBlock** [comment]
[22621.232_arm_221018.part1.rar](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/iso/22621.232_arm_221018.part1.rar)
[22621.232_arm_221018.part2.rar](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/iso/22621.232_arm_221018.part2.rar)

**Comment by hudsonhu** [comment]
It works, thank you!

**Comment by WhatTheBlock** [comment]
@hudsonhu 
Hi! Can you provide the running screen of 22621.232_arm_221018 for the release page? Thanks!
In the future, I will continue to update the ARM64 version, I hope you can help test.

**Comment by hudsonhu** [comment]
Sure, I will be happy to participate in the testing of the new version.
<img width="1440" alt="10" src="https://user-images.githubusercontent.com/48980398/197098216-d28909c4-f0c8-4489-8944-35ba6689b9e5.png">

**Comment by WhatTheBlock** [comment]
@hudsonhu 

New version is out!
[22623.870_arm_221021](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w11.arm.221021)

This version adds more performance optimizations, but it is not sure that it will work properly.

**Comment by hudsonhu** [comment]
@WhatTheBlock Tested with Microsoft Store Apps, and a few traditional exes. Works fine!
<img width="1920" alt="870" src="https://user-images.githubusercontent.com/48980398/197224679-7a5ee859-5a80-4cf0-a3fc-62b1c57dfd0f.png">

**Comment by WhatTheBlock** [comment]
I'm not sure if Win10 will perform better on ARM64.

[17763.3534_arm_221025](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w10.arm.221025)

**Comment by hudsonhu** [comment]
It stucks and crashes into blue screen, when "Copying Windows Files 0%" in installation. @WhatTheBlock 

<img width="1136" alt="2022-10-25 16 59 46" src="https://user-images.githubusercontent.com/48980398/197731747-783a4f57-d374-46b5-b1ec-b4377c0fa63f.png">

**Reply by WhatTheBlock** [reply]
Bugfix version uploaded.
[Link](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/iso/17763.3534_arm_221025_f.iso)

**Comment by hudsonhu** [comment]
New image still crashes.
<img width="1136" alt="copying" src="https://user-images.githubusercontent.com/48980398/197775764-9964d751-00c2-43c2-9e10-2bab1997a8e6.png">

About 8 sec later,
<img width="1136" alt="crash" src="https://user-images.githubusercontent.com/48980398/197775845-5cb321b2-70e8-4261-b8ae-7728f93e0ba5.png">

**Reply by hudsonhu** [reply]
Tried removing the TPM chip in VM. Still getting the error.

**Reply by WhatTheBlock** [reply]
It seems that there may be a compatibility issue between Win10 ARM64 and the virtual machine, as only Windows Setup will be executed at this stage. The bugfix version I replaced boot.wim with the original one still happens, which is unreasonable.

**Reply by WhatTheBlock** [reply]
Have you tried [UTM](https://github.com/utmapp/UTM)? I don't know if there is any difference in Windows compatibility between this and PD.

**Reply by hudsonhu** [reply]
Yes, it seems like a PD issue. I'll try it on vmware for Apple Silicon when I have time.
